### PR TITLE
Splitting iRacing.cs

### PR DESCRIPTION
**Pull request by photon1503**
- New classes for Rotary (inCarMenu, PitMenu)
- New class for SmoothGear
- New class for Acceleration stopwatch

Let me know if this works for you

### CODE REVIEW

File: Sections/Rotary.cs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+namespace DahlDesign.Plugin.Categories
+{
+    /// <summary>
+    /// Rotary button class
+    /// </summary>
+    public class Rotary : SectionBase
+    {
+
+        public Rotary(DahlDesign dahlDesign) : base(dahlDesign) { }
```
Comment (andreasdahl1987) [inline Sections/Rotary.cs:11]: When using increments/decrements the menu goes 1-12, but using any of the specific bindings brings it to 13, and it is locked there.

File: Sections/SmoothGear.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+using SimHub.Plugins;
+
+namespace DahlDesign.Plugin.Categories
+{
+    /// <summary>
+    /// Smooth Gear
+    /// </summary>
+    public class SmoothGear : SectionBase
```
Comment (andreasdahl1987) [inline Sections/SmoothGear.cs:8]: Tested and working, noticed error that would prevent a DDC wheel from getting an "N" in this prop when neutral is active on the wheel. Made a commit to fix it.

File: Sections/Acceleration.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
+using System;
+using SimHub.Plugins;
+
+namespace DahlDesign.Plugin.Categories
+{
+    /// <summary>
+    /// Acceleration stopwatch
+    /// </summary>
+    public class Acceleration : SectionBase
```
Comment (andreasdahl1987) [inline Sections/Acceleration.cs:9]: Tested and working :)

**Review by andreasdahl1987 — COMMENTED** [review]
Have had a look

File: Sections/Rotary.cs (hunk 1 above)
Comment (andreasdahl1987) [reply Sections/Rotary.cs:11]: Ok, fixed in the latest commit. Don't know why that fixed it thb, might be something about how these actions are built?

File: .vscode/tasks.json [hunk 4]
```diff
@@ -6,8 +6,8 @@
         {
             "label": "build",
             "type": "shell",            
-            "command": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\Msbuild\\Current\\Bin\\msbuild",
-            //"command": "msbuild",
+            //"command": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\Msbuild\\Current\\Bin\\msbuild",
```
[the remark this answers is not in the document]
Comment (photon1503) [reply .vscode/tasks.json:9]: Only if they use VS Code to build the solution. "msbuild" should be somewere in the path

**Pull request merged**